Drafts/scs-XXXX-v0-naming-conventions.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Recommended naming for domains/groups/roles/project when onboarding new customers
+type: _Standard | Decision Record_
+status: Draft
+track: IAM
+---
+
+<!---
+This is a template striving to provide a starting point for
+creating a standard or decision record adhering to scs-0001.
+Replace at least all text which is _italic_.
+See https://github.com/SovereignCloudStack/standards/blob/main/Standards/scs-0001-v1-sovereign-cloud-standards.md
+--->
+
+## Introduction
+
+When CSPs try to enroll a new customer they encounter themselves in
+a situation where they have to choose names for the openstack domain,
+project and user.
+
+There might also exists some group or roles from the customers point of
+view, like `domain admin` or `project admin` not related at all with
+openstack.
+
```

Comment:
Maybe we should move this topic out of the introduction and turn it instead into a clarification of the "PS approach to naming" below.

```suggestion
```